keps/sig-network/20190603-EndpointSlice-API.md

```diff
@@ -59,81 +59,116 @@ type EndpointSlice struct {
     metav1.TypeMeta `json:",inline"`
     // OwnerReferences should be set when the object is derived from a k8s
     // object.
-    // The object labels may contain the following key:
+    // The object labels may contain the following keys:
```

Comment:
We should change "may contain" to "may include", and we should add "...from which the EndpointSlice is derived.  EndpointSlices which are not associated with a Service should not use this key."

Reply:
Done, thanks!